src/app/shared/components/smart-search/smart-search.component.html

```diff
@@ -14,11 +14,11 @@
     </mat-autocomplete>
     <div matSuffix class="d-flex" *ngIf="searchTxtFieldValue?.trim()?.length > 2">
         <ng-container *ngIf="!isAutolistLoading?.value">
-            <button mat-icon-button (click)="search()">
+            <button mat-icon-button (click)="search()" class="searchButton">
                 <mat-icon>search</mat-icon>
             </button>
-            <button mat-icon-button (click)="clearSearchTxt()">
-                <mat-icon>close</mat-icon>
```

Comment:
This smart search is also used for handling setting role issuer for `Issuer Type = Role`. Now it shows possibility to add, but we can't do that -> it just clears input. So this change made a regression.